sdks/unreal/Agones/Source/Agones/AgonesHook.cpp

```diff
@@ -34,15 +40,24 @@ FAgonesHook::FAgonesHook()
 	, ReadySuffix(FString(TEXT("/ready")))
 	, HealthSuffix(FString(TEXT("/health")))
 	, ShutdownSuffix(FString(TEXT("/shutdown")))
+	, SetLabelSuffix(FString(TEXT("/metadata/label")))
+	, SetAnnotationSuffix(FString(TEXT("/metadata/annotation")))
+	, GetGameServerSuffix(FString(TEXT("/gameserver")))
 {
 	Settings = GetDefault<UAgonesSettings>();
 	check(Settings != nullptr);
 
-	UE_LOG(LogAgonesHook, Log, TEXT("Initialized Agones Hook, Sidecar address: %s, Health Enabled: %s, Health Ping: %f, Debug: %s")
+	uint32 RetryLimitCount = Settings->RequestRetryLimit;
+	HttpRetryManager = MakeShared<FHttpRetrySystem::FManager>(
+		FHttpRetrySystem::FRetryLimitCountSetting(RetryLimitCount),
+		FHttpRetrySystem::FRetryTimeoutRelativeSecondsSetting());
+
+	UE_LOG(LogAgonesHook, Log, TEXT("Initialized Agones Hook, Sidecar address: %s, Health Enabled: %s, Health Ping: %f, Debug: %s, Request Retry Limit: %d")
 		, *SidecarAddress
 		, (Settings->bHealthPingEnabled ? TEXT("True") : TEXT("False"))
 		, Settings->HealthPingSeconds
-		, (Settings->bDebugLogEnabled ? TEXT("True") : TEXT("False")));
+		, (Settings->bDebugLogEnabled ? TEXT("True") : TEXT("False"))

```

Comment:
@WVerlaek we discussed this yesterday but adding here for posterity!

In a subsequent PR we should probably ditch `bDebugLogEnabled` and instead just make use of the log levels in Unreal's logging system, which would:
a) Allow developers to change the logging behaviour through .ini configuration files, and on-the-fly through console commands. As an example, through the in-game console (or remotely using the "messaging" system), you could run the command `log LogAgonesHook error` to suppress any logs with a severity less than `Error`
b) Make the plugin more consistent with the rest of the engine
c) Remove some code

Reply:
Agree, standardize on the engine and how UE4 enginers' work would be fantastic in a follow-up. 

Mentioned in the past that blueprint support would be fantastic.